sdk/nodejs/src/__mocks__/axios.ts

```diff
@@ -0,0 +1,98 @@
+module.exports = {
+  request: jest.fn((req) => {
```

Comment:
it seems like you should be using Nock for this

Reply:
Switched to using `msw` instead of Nock... cleaner definitions and easier to manage